File: C#/EntityFramework_ORM/DAL/Entities/Region.cs
using DAL.Interfaces;
using System;
using System.Collections.Generic;

namespace MyApp.DAL.Entities;

public partial class Region : IEntity
{
    public int RegionID { get; set; }

    public string RegionDescription { get; set; } = null!;

    public virtual ICollection<Territory> Territories { get; set; } = new List<Territory>();
}

File: Python/SQLAlchemy_ORM/dal/entities/region.py
from sqlalchemy import Column, Integer, String
from sqlalchemy.orm import relationship
from dal.data.base import Base

class Region(Base):
    __tablename__ = 'Region'

    RegionID = Column(Integer, primary_key=True)
    RegionDescription = Column(String(50))

    territories = relationship('Territory', back_populates='region')